### DISCUSSION TITLE
Make it possible to change default corner controls line width.

### DISCUSSION THREAD

**Discussion by Botelho31** [fabricjs/fabric.js#10279] [Ideas]
I happen to have a specific usecase where i need to scale the entire standard controls and happen to notice that i can scale everything BUT the corner stroke width. Wouldn`t it be a nice and simple addition? i already have the code done but the PR formating said i should open a discussion first.

**Comment by asturur** [comment]
I think that is already possible, is a weird-named parameter:

https://fabricjs.com/api/interfaces/fabricobjectprops/#borderscalefactor

Try if this is what you are looking for.

I ll improve the wording

**Comment by Botelho31** [comment]
I am already using that! It changes the width of the outline of the image, but the actual corner controls stay the same line width.

**Reply by asturur** [reply]
Could be a bug introduced with refactors. they are supposed to be same.

**Reply by asturur** [reply]
It is broken since v4 indeed in v3 the behavior is how i remembered

**Reply by Botelho31** [reply]
Should i open a PR to fix it?

**Reply by asturur** [reply]
I already did open one https://github.com/fabricjs/fabric.js/pull/10283
